melange-react-intl-ppx.opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+version: "git"
+maintainer: "Ahrefs"
+authors: "Ahrefs"
+dev-repo: "git+https://github.com/ahrefs/bs-react-intl-ppx.git"
+license: "MIT"
+homepage: "https://github.com/ahrefs/bs-react-intl-ppx"
+bug-reports: "https://github.com/ahrefs/bs-react-intl-ppx/issues"
```

Comment:
I'd probably point these to `melange-` already, so there are less things we have to remember to update later on.

Reply:
Good point. Updated:
https://github.com/ahrefs/bs-react-intl-ppx/blob/9e7418251f817c0a44d47820f923285702de54dd/melange-react-intl-ppx.opam#L7-L8